app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1 class='text-center mt-5'><%= @recipe.name %></h1>
+
+<section class="text-center mt-5">
+  <%= render @recipe %>
+  <div class="container my-5 ">
+    <div class="row">
+      <div class="d-flex justify-content-end align-content-center gap-5">
+        <%= button_to "Generate Shopping List", generate_shopping_list_recipe_path(@recipe), method: :post, class: "btn btn-primary" %>
+
+        <% if current_user == @recipe.user %>
+          <%= link_to "Add Ingredient", new_recipe_recipe_food_path(@recipe), class: "btn btn-success" %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+  <div class='container'>
```

Comment:
- [ ] @Team Great work implementing the changes requested, however, you still haven't changed the layout to match the wireframe layout as previously requested

Given Layout         |  Your Layout   
:-------------------------:|:-------------------------:
![image](https://github.com/martinkarugaba/recipe_app/assets/61408860/0ab993f1-7903-4afc-a240-becca2b33cfa) |  ![image](https://github.com/martinkarugaba/recipe_app/assets/61408860/b3e6289e-c85f-4ab8-bc8c-3564fc7401de)